everest

```diff
@@ -509,34 +531,43 @@ do_reset ()
   done
 }
 
+run_nuget () {
+  # This function is called in a subshell in a test context (via possibly_log)
+  # which means that -e is not inherited. Explicit failures here.
+  cmd=$1
+  nuget restore vale/tools/Vale/src/packages.config -PackagesDirectory vale/tools/FsLexYacc $cmd || return 1
+
+}
+
 run_scons () {
   # This function is called in a subshell in a test context (via possibly_log)
   # which means that -e is not inherited. Explicit failures here.
-  # OS-independnt scons. Pass $1 to scons.
+  # OS-independent scons. Pass $1 to scons.
   cmd=$1
   if is_windows; then
     VARS=$(cygpath -d "$VS_BIN\\..\\vcvarsall.bat")
-    DIR=$(cygpath -d "$(pwd)"/Spartan)
+    DIR=$(cygpath -d "$(pwd)"/vale)
     CMD=$(cygpath -u "$COMSPEC")
     $CMD /C "cd $DIR && $VARS amd64 && scons.bat $cmd" || return 1
   else
-    scons -C Spartan $cmd || return 1
+    scons -C vale $cmd || return 1
```

Comment:
ibid... don't need the || return 1 when this is the last command of the function